suite-native/module-connect-device/src/components/PinFormProgress.tsx

```diff
@@ -22,17 +23,26 @@ const enteredDigitsStyle = prepareNativeStyle(utils => ({
 
 export const PinFormProgress = () => {
     const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
     const { watch } = useFormContext();
 
     const pinLength = watch('pin').length;
 
-    if (!pinLength) return <Text variant="titleSmall">Enter PIN</Text>;
+    if (!pinLength)
+        return <Text variant="titleSmall">{translate('moduleConnectDevice.pinScreen.title')}</Text>;
 
     if (pinLength > MAX_DIGITS_DISPLAYED_AS_DOTS)
         return (
-            <Text style={applyStyle(enteredDigitsStyle)} color="textSubdued">
-                Entered <Text variant="highlight">{pinLength}</Text> digits
-            </Text>
+            <Box flexDirection="row" style={applyStyle(enteredDigitsStyle)}>
```

Comment:
Could not this whole `Box` be just a one translation string, if you would apply[ Intl rich text formatting](https://formatjs.io/docs/react-intl/components/#rich-text-formatting)? If the `moduleConnectDevice.pinScreen.entered` and `moduleConnectDevice.pinScreen.digits` are used only in this place, then a single translation ID would be sufficient.

Reply:
I also need to transform the styles based on different parts of the text so decided to just keep it simple like this. If it was simple sentence, then yes, this would make sense.

Reply:
Rich allows styled formatting. But whatever, It's not that important.